tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+oc create namespace kubevirt
```

Comment:
consider please
NAMESPACE=${KUBEVIRT_INSTALLED_NAMESPACE:-kubevirt}
as other script use

Reply:
@enp0s3 , this line seems redundant, right? Why would we create a namespace named "kubevirt"?

Reply:
@EdDev You are right, I've missed that one. It can be removed